src/components/AposCard.vue

```diff
@@ -0,0 +1,33 @@
+<!--
+    @author JanGut
+    @date 2020-03-08
+-->
+<template>
+    <div class="apos-card">
+        <h3 class="apos-card-title">{{ this.CardTitle }}</h3>
```

Comment:
```suggestion
        <h3 class="apos-card-title">{{ this.cardTitle }}</h3>
```
cardTitle is spelled lowercase in props